mitmproxy/net/tcp.py

```diff
@@ -743,12 +653,12 @@ def convert_to_ssl(self, sni=None, alpn_protos=None, **sslctx_kwargs):
 
     def makesocket(self):
```

Comment:
Does the cuckoo sandbox team use this API? I would like to limit the API changes if possible.
Or do you know how they use it?